src/css/controls/imports/icons.less

```diff
@@ -118,3 +118,24 @@
     }
 }
 // stylelint-enable indentation
+.jwplayer.jw-breakpoint--1 {
```

Comment:
should be in display.less I think

Reply:
hm even though it's icon styling? i'm not familiar with the file breakdown - looks like there are some restylings in `display` related to breakpoint but not all and if i was looking for where i could change icons display i would look here but don't want to break conventioin!

@robwalch any input here?